libdnf/comps/comps.cpp

```diff
@@ -2,8 +2,21 @@
 #include <libdnf/comps/group/group-private.hpp>
 #include <libdnf/utils/xml.hpp>
 
+extern "C" {
+#include <solv/pool.h>
+#include <solv/repo.h>
+#include <solv/repo_comps.h>
+#include <solv/repodata.h>
+}
+
 #include <cstring>
 
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <libsmartcols/libsmartcols.h>
```

Comment:
I suppose this is a temporary include.
Libdnf is not supposed to link with libsmartcols; such code should be part of libdnf-cli.